Cat2Cat.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "Cat2Cat"
-  s.version      = '1.4.1'
+  s.version      = '1.5.0'
```

Comment:
Given that this'll break existing-working Swift 1.2 projects, I think this would have to be 2.0.0...